app/components/feeds/JoinFeedCard.tsx

```
@@ -0,0 +1,148 @@
+        {feed.description && (
+          <div className={styles.descriptionContainer}>
+            <ShowMore
+              ref={showMoreRef}
+              lines={3}
+              more={
+                <>
+                  {" ... "}
+                  <button
+                    className={styles.readMoreButton}
+                    onClick={toggleLines}
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)